MpsAllocatorSample/Program.cs

```diff
@@ -203,6 +203,24 @@ static async Task RequestMultiplayerServer()
             string region = Console.ReadLine();
             req2.PreferredRegions = new List<string>() {region};
             req2.SessionId = Guid.NewGuid().ToString();
+            // InitialPlayers is the list of the initial players of the game
```

Comment:
The following is the documentation comment in PF code and I feel it is more informative, we can copy paste it here:

"Initial list of players (potentially matchmade) allowed to connect to the game. This list is passed to the game server when requested (via GSDK) and can be used to validate players connecting to it."